examples/graphiql-parcel/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "graphiql-parcel-example",
+  "version": "1.0.0",
+  "license": "MIT",
+  "description": "GraphiQL Parcel Example",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+    "start": "parcel src/index.html -p 8080",
+    "build": "parcel build src/index.html --public-url /"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "dependencies": {
+    "@types/react": "^16.9.34",
+    "@types/react-dom": "^16.9.6",
+    "graphiql": "file:../../packages/graphiql",
```

Comment:
can you make this `file:packages/graphiql` instead? i need to change all the examples to do this

Reply:
Made the required change but it throws an error when we build it.

```
error Package "graphiql" refers to a non-existing file '"/home/runner/work/graphiql/graphiql/examples/graphiql-parcel/packages/graphiql"'.
15
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

Reply:
@acao can you look into this

Reply:
@daemon1024 pointed out that this didn't work, i had it backwards, my bad.